uiowa/uiowa_profile
# Main navigation

This PR addresses #43 by:

- creating and placing a Superfish menu block for top-level links
- creating and placing a menu_block block for level 2+ menu links
- assigning permissions for editors & webmasters to manage the Main Navigation menu

**Note:** As a follow-up issue, the block placements provided here will need to be changed once we have a custom theme set as the default.

**To test this PR:**
1. Check out the `superfish` branch of uiowa801.
2. Run `composer install`.
3. Run `drush si uiowa` for some site.
4. Add some content to the Main menu.
5. Verify that:
    - editors & webmasters can add Basic Pages to the Main menu from the node add/edit form
    - editors & webmasters can manage only the Main menu at admin/structure/menu
    - the superfish block and menu_block block display
